setup/Swix/Microsoft.FSharp.Compiler/Files.swr

```diff
@@ -0,0 +1,34 @@
+use vs
+
+package name=Microsoft.FSharp.Compiler
+        version=$(FSharpPackageVersion)
+
+folder "InstallDir:Common7\IDE\CommonExtensions\Microsoft\FSharpCompiler"
+  file source="$(BinariesFolder)\net40\bin\fsc.exe" vs.file.ngen=yes
```

Comment:
We had some problems about ngen priority with Roslyn - is this right?

Reply:
I'll talk to some folks on the compiler team to see what issues they had.